src/mpid/ch4/shm/xpmem/xpmem_control.h

```diff
@@ -22,13 +24,19 @@
 MPL_STATIC_INLINE_PREFIX int MPIDI_XPMEM_ctrl_send_lmt_ack_cb(MPIDI_SHM_ctrl_hdr_t * ctrl_hdr)
 {
     int mpi_errno = MPI_SUCCESS;
-    MPIR_Request *sreq = (MPIR_Request *) ctrl_hdr->xpmem_slmt_ack.sreq_ptr;
+    MPIR_Request *req = (MPIR_Request *) ctrl_hdr->xpmem_slmt_ack.sreq_ptr;
 
```

Comment:
Is this change unnecessary ?

Reply:
`req` is just for the reminder that this function is used by both sender and receiver not only by sender.

Reply:
Why is this function used by both sender and receiver ? It should be called only by sender. If the receiver needs an ack callback, it should be defined separately.

Reply:
sure, I will separate them into two functions.